LSPS1/extensions/refunds/LSPS1.refunds.yaml

```diff
@@ -0,0 +1,154 @@
+openapi: 3.0.3
+info:
+  description: |
+    API documentation for channel opening request from a Lightning Network Service Provider (LSP). Refunds.
+    
+  version: "1"
+  title: "LSP Spec 1"
+tags:
+- name: "Refund"
+  description: "Extension: `refunds`"
+
+
+paths:           
+  /lsp/channels/{id}/refund:
```

Comment:
Regarding privacy, would it be better to put the channel id in the request itself? Someone sniffing could see the channel id and request the information itself, because there is no form of authentication (I imagine authentication would be hard to implement too). If the parameters are in the request body, and transport security is used, at least only the LSP is able to read the channel information.
That would break the REST setup of the proposal though. It would become more of an RPC style spec with POST requests for everything.

Reply:
Good point especially as the id kind of acts as a secret for refunds.

Reply:
Are you sure though? https://stackoverflow.com/a/499594

Reply:
Closing this discussion as I think privacy is given (see stackoverflow) and I didn't get any more inputs from you @JssDWt.